index.d.ts

```diff
@@ -2,38 +2,61 @@ declare module "prism-react-renderer" {
   import * as React from "react";
 
   type Language =
+    | "plain"
+    | "plaintext"
+    | "text"
+    | "txt"
+    | "extend"
+    | "insertBefore"
+    | "DFS"
     | "markup"
+    | "html"
+    | "mathml"
+    | "svg"
+    | "xml"
+    | "ssml"
+    | "atom"
+    | "rss"
```

Comment:
I may be mistaken here, but we only load a subset of all the languages provided by Prism. See the code here: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js

Reply:
But... why limit the feature set of the product you're wrapping? Perhaps adding a config to limit language support if the implementer needs to shave a few bytes off the deployment, but it seems weird and arbitrary to force developers using this component to manually re-add support.